docs/archives.md

```diff
@@ -144,6 +155,83 @@ export default withArchive( posts, state => state.posts, 'blog' )( Blog );
 ```
 
 
+### Manual Pagination
+
+For manual pagination, use the `withPagedArchive` HOC instead of `withArchive`. This allows you to manually specify the page of the archive to load, and ignores Repress's internal page counter. Additionally, only the current page of items is passed to your component.
+
+This HOC passes the same props as `withArchive`, with the following differences:
+
+* `posts` (`object[]`): A list of objects on the given page of the archive.
+* `page` (`Number`): The current page being viewed.
+* `totalPages` (`Number`): Total number of pages available for the archive.
+
+The archive page to load should be passed via the `page` prop to your component. To override this behaviour, you can pass a `getPage` function in the `options` parameter to the HOC:
+
+* `getPage` (`Function`: `object => Number`): Map passed props to the page number. By default, this is `props => props.page`.
+
+`onLoadMore` is automatically called for you when the page changes. You can use `this.props.hasMore` to determine whether to show navigation to the next page, and you can check `this.props.page > 1` to determine whether to show navigation to the previous page.
+
+For example, the following component renders a simple list of posts with navigation to browse back and forth:
+
+```js
+function Blog( props ) {
+	const { hasMore, loading, loadingMore, page, posts } = props;
+
+	if ( loading ) {
+		return <div>Loading…</div>;
+	}
+
+	if ( ! posts ) {
+		return <div>No posts</div>;
+	}
+
+	return <div>
+		<ol>
+			{ posts.map( post =>
+				<li key={ post.id }>{ post.title.rendered }</li>
+			) }
+		</ol>
+
+		{ loadingMore ?
+			<p>Loading more…</p>
+		: (
+			<div>
+				{ hasMore ? (
+					<button
+						type="button"
+						onClick={ () => props.onOlder() }
```

Comment:
Can this just be `props.onOlder`

Reply:
Yes, but copying the pattern from above.